db/migrate/20241202152952_change_capital_disregards_index.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeCapitalDisregardsIndex < ActiveRecord::Migration[7.2]
+  disable_ddl_transaction!
+
+  def change
+    remove_index(:capital_disregards, name: :index_capital_disregards_on_legal_aid_application_id_and_name)
+    add_index(:capital_disregards, [:legal_aid_application_id, :name, :mandatory], unique: true, algorithm: :concurrently)
```

Comment:
not a blocker but might want to consider [adding the index concurrently](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/7469/commits/84cf09096f5f20b0767238873d861ad133d7194e)